CoreScriptsRoot/CoreScripts/Topbar.lua

```diff
@@ -713,7 +700,7 @@ local function CreateSettingsIcon(topBarInstance)
 			settingsActive = false
 		end
 
-		MenuModule:ToggleVisibility(settingsActive)
+		MenuModule:ToggleVisibility(settingsActive, nil, nil, usingGamepad)
```

Comment:
Oh, this should probably go. Ben and I talked about this; the last parameter isn't being used anymore.